content/docs/installation/contents.lr

```diff
@@ -32,9 +32,9 @@ the installation is a bit more involved.
 
 You need to make sure you have the following software installed on your computer:
 
-* Python 2.7 or above (also `python-dev`, `libssl-dev` and
-  `libffi-dev` is required on Ubuntu)
-  `sudo apt-get install python-dev libssl-dev libffi-dev`
+* Python 3 is recommended (but 2.7 is also supported)
+  On Ubuntu `python3-dev`, `libssl-dev` and `libffi-dev` are also required
```

Comment:
Do we know which dependency that requires/required this? I can see the text was there from before, but a lot of dependencies bundle maylinux wheels nowadays so dev dependencies aren't necessary anymore.

Reply:
i have no idea. it's not in the scope of this PR though, maybe add a new issue.

Reply:
by the way, I had to do https://github.com/lektor/lektor-website/blob/9afb4b00a37ef8d29cc8c5803c8f43d09812a448/assets/installer.py#L231 on windows, because `cryptography` would fail to install, with an error about "this is a PEP 517 wheel".  So some building is involved.